packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-page-list/account-page-list.tsx

```diff
@@ -0,0 +1,327 @@
+import {
```

Comment:
Have you checked how this all interacts with the new page loading skeleton stuff?

Reply:
I had, but because I was using a `useEffect` to derive the state I had to create an `initialLoading` state variable to prevent a jump from the skeleton pages to the actual page list: fa202ed8492032f750d96672acc20b43a68f71fa

This was not a problem anymore after f39a7ce8a6261d522bec2d935b1475e02c9ab675 so I reverted to using `pagesLoading` to check what to display (skeleton or pages) in b6ddd7f91cd6fff5e272e3f9e46cd4e2bb4e8535